backend/app/crud/crud_content.py

```
@@ -398,13 +410,46 @@ def update_content_item_sync(
     return db_content_item
 
 
-def delete_content_item_sync(session: Session, id: uuid.UUID) -> ContentItem | None:
-    db_content_item = get_content_item_sync(session, id)
-    if db_content_item:
-        session.delete(db_content_item)
+def delete_content_item_sync(session: Session, id: uuid.UUID, user_id: uuid.UUID) -> bool:
+    """
+    DELETE CASCADE delete content item and all related records
+    """
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)